gradle/libs.versions.toml

```diff
@@ -1,41 +1,43 @@
 [versions]
 ## SDK Versions
 minSdk = "21"
-targetSdk = "33"
-compileSdk = "33"
+compileSdk = "34"
 
 # Dependencies
-kotlin = "1.9.0"
+kotlin = "1.9.10"
 binaryCompatability = "0.13.2"
 
-androidx-core = "1.8.0"
-androidx-appcompat = "1.5.1"
+androidx-core = "1.12.0"
+androidx-appcompat = "1.6.1"
 androidx-constraintLayout = "2.1.4"
-androidx-navigationFragment = "2.5.2"
-androidx-navigationUI = "2.5.2"
+androidx-navigationFragment = "2.7.2"
+androidx-navigationUI = "2.7.2"
 androidx-coordinatorLayout = "1.2.0"
-android-material = "1.6.1"
-android-gradle-plugin = "7.2.2"
-android-library = "7.2.2"
 
-google-services = "4.3.10"
+android-gradle-plugin = "8.1.1"
+android-test-runner="1.5.2"
 
-stately = "2.0.0-rc3"
-testhelp = "0.6.5"
+google-services = "4.3.15"
 
-crashkios = "0.8.2"
-bugsnag = "5.26.0"
-
-firebase-bom = "28.4.0"
-
-koin = "3.4.2"
-coroutines = "1.6.4"
-roboelectric = "4.8.2"
-buildConfig = "2.1.0"
-mavenPublish = "0.22.0"
+stately = "2.0.0"
+testhelp = "0.6.9"
 
-android-junitTest = "1.1.3"
+crashkios = "0.8.2"
```

Comment:
Also whatever the new release is, probably 0.8.3

Reply:
will wait for CrashKiOS release before updating and merging this PR. Thanks